src/dandisets_linkml_status/cli/__init__.py

```diff
@@ -0,0 +1,73 @@
+import logging
+from typing import Annotated
+
+import typer
+from dandi.dandiapi import DandiAPIClient
+from pydantic2linkml.cli.tools import LogLevel
+
+from dandisets_linkml_status.cli.models import DandisetValidationReport
+from dandisets_linkml_status.cli.tools import DandisetLinkmlValidator, pydantic_validate
+
+logger = logging.getLogger(__name__)
+app = typer.Typer()
+
+
+@app.command()
+def main(
+    *,
+    include_unpublished: Annotated[
+        bool, typer.Option("--include-unpublished", "-u")
+    ] = False,
```

Comment:
to not wait for querying all of them, please add an option `--dandiset` for a dandiset regex to operate just on a selection of those . We have similar one (although typically as argument, not option) in https://github.com/dandi/backups2datalad/blob/main/src/backups2datalad/__main__.py#L243 .